dali/pipeline/operator/op_spec.h

```diff
@@ -81,6 +79,14 @@ class DLL_PUBLIC OpSpec {
    * @brief Sets the name of the Operator.
    */
   DLL_PUBLIC inline void set_name(const string &name) {
+    // Due to the fact that External Source existed in DALI as two entities we need to have a place
+    // where we merge it back into one. "ExternalSource" had special handling for serialization,
+    // so we go with the _ExternalSource that is now used.
+    if (name == "ExternalSource") {
+      name_ = "_ExternalSource";
+      schema_ = SchemaRegistry::TryGetSchema("_ExternalSource");
+      return;
+    }
```

Comment:
```suggestion
    if (name == "ExternalSource")
      name = "_ExternalSource";
```
Anyway, I somehow think that it would be better to do it in TryGetSchema.

Reply:
The problem is, without renaming the spec somewhere, we get incosistencies and _ExternalSource can still appear (for example after deserialization).